tests/unit/src/test/scala/tests/decorations/SyntheticDecorationsLspSuite.scala

```diff
@@ -647,15 +648,6 @@ class SyntheticDecorationsLspSuite extends BaseLspSuite("implicits") {
       _ <- server.didChange("a/src/main/scala/Main.scala") { text =>
         "}\n" + text
       }
-      _ = assertNoDiff(
```

Comment:
Because of parsing error `trees` holds the old version of the file, so positions don't match. I don't know if we can make it work correctly for not parsing files

Reply:
We could use the edit distance same as we do with semanticdb.